index.html

```diff
@@ -0,0 +1,54 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Landing page</title>
+    <link rel="stylesheet" href="./resources/css/style.css"/>
+</head>
+<body>
+    <nav>
+        <div class="heading">
+            <img class="logo"
+            src="./resources/images/36.jpg" alt="logo image"/>
+             <h2 class="nav_heading">FavCrunchies</h2>
+        </div>
+        <div class="right_cnt">
+        <ul>
+            <li><a href="/">Home</a></li>
+            <li><a href="/profile">Profile</a></li>
+            <li><a href="/project">Project</a></li>
+            <li><a href="/contact">Contact</a></li>
+            <li><a href="/about">About</a></li>
+        </ul>
+        <button class="btn">Sign up</button>
+        </div>
+    </nav>
+<main>
+
+    <div class="content">
+
+        <h1>Grab Your FavCrunchies</h1>
+        <P class="description">we make the best cakes, cookies and extraordinry milkshakes...we are the best!!!</P>
+   <button class="pri_btn">
+    REGISTER
+   </button>
+    </div>
+    <img class="image" src="./resources/images/47.jpg" alt="">
+
+</main>
+    
+    
+    
+    
+    
+    
+    
+    <footer>ONWURAH OKECHUKWU JOSEPH
+
```

Comment:
Remove this line and the footer closing tag can be on same line with the opening tag since the contents are not much